packages/modal/storybook/Modal.js

```diff
@@ -7,6 +7,11 @@ import styled from 'styled-components'
 const StyledModal = styled(Modal)`
   height: ${props => props.height};
 `
+const CUSTOM_ANIMATION = transitionstate => `
```

Comment:
Consistent camel case for variable name?

```suggestion
const CUSTOM_ANIMATION = transitionState => `
```